modules/public/public.go

```diff
@@ -23,8 +23,8 @@ type Options struct {
 	CorsHandler func(http.Handler) http.Handler
 }
 
-// AssetsURLPathPrefix is the path prefix for static asset files
-const AssetsURLPathPrefix = "/assets/"
+// WebPublicDirName is the directory name for public(static) asset files. It may be set to the build time by `static.go`
+var WebPublicDirName = "public-dynamic"
```

Comment:
could we do `/assets/dynamic` instead - if we just change reserved words for users each time a new version pops up that just break stuff!

Reply:
(I find breaking stuff for no reason is just nonsense)

Reply:
If we use `~` char, we do not need to reverse any names anymore. See the comment below.

And I prefer to make the names consistent, we shouldn't use `/assets` for `GITEA_CUSTOM/public`